### PR TITLE
Unify Android and standard Docker image

### LINKED ISSUE #3336: oak_tensorflow_service compilation failing in Docker
When running

```
./scripts/docker_run cargo build --package=oak_tensorflow_service
```

I get the following error

```
  DEBUG: /root/.cache/bazel/_bazel_root/eab0d61a99b6696edb3d2aff87b585e8/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:125:14: rbe_default not using checked in configs; Bazel version 5.3.1 was picked/selected but no checked in config was found in map {"0.20.0": ["8.0.0"], "0.21.0": ["8.0.0"], "0.22.0": ["8.0.0", "9.0.0"], "0.23.0": ["8.0.0", "9.0.0"], "0.23.1": ["8.0.0", "9.0.0"], "0.23.2": ["9.0.0"], "0.24.0": ["9.0.0"], "0.24.1": ["9.0.0"], "0.25.0": ["9.0.0"], "0.25.1": ["9.0.0"], "0.25.2": ["9.0.0"], "0.26.0": ["9.0.0"], "0.26.1": ["9.0.0"], "0.27.0": ["9.0.0"], "0.27.1": ["9.0.0"], "0.28.0": ["9.0.0"], "0.28.1": ["9.0.0"], "0.29.0": ["9.0.0"], "0.29.1": ["9.0.0", "10.0.0"], "1.0.0": ["9.0.0", "10.0.0"]}
  Loading: 0 packages loaded
  Analyzing: target //cc/tflite_micro:tflite-micro (1 packages loaded, 0 targets configured)
  ERROR: /workspace/WORKSPACE:296:23: fetching android_ndk_repository rule //external:androidndk: Either the path attribute of android_ndk_repository or the ANDROID_NDK_HOME environment variable must be set.
  ERROR: Analysis of target '\''//cc/tflite_micro:tflite-micro'\'' failed; build aborted: Either the path attribute of android_ndk_repository or the ANDROID_NDK_HOME environment variable must be set.
  INFO: Elapsed time: 20.450s
  INFO: 0 processes.
  FAILED: Build did NOT complete successfully (5 packages loaded, 6 targets configured)
  FAILED: Build did NOT complete successfully (5 packages loaded, 6 targets configured)
  thread '\''main'\'' panicked at '\''Failed to run bazel build: exit status is exit status: 1'\'', oak_tensorflow_service/build.rs:52:9
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace'
```

@ipetr0v could you take a look? Also maybe we should consider merging the Android and dev Docker image now that they both work with Java 11.

### PR DESCRIPTION
**Pull request by tiziano88**
Everything now works on JDK11, so a single image allows us to run all
builds and tests at the same time, which will be useful as we continue
working on the Java client.

This means that the image is now somewhat larger, but it is much easier to make sure that everything stays in sync and keeps working correctly, and also test things locally.

Also cleanup and refactor the Bazel WORKSPACE file to work with the latest version of Bazel.

Fix #3336 #3326

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -251,7 +237,12 @@ RUN cargo install --version=${wizer_version} wizer --all-features
 
 # Install crosvm.
 # We're not interested in most of the features in crosvm (e.g. wayland support), but GDB support would be nice.
-RUN cargo install --git https://chromium.googlesource.com/chromiumos/platform/crosvm/ --rev 31f04e92709980a4ffc56b1631f8b4be437cc2fe crosvm --no-default-features --features gdb
+RUN cargo install \
+  --git=https://chromium.googlesource.com/chromiumos/platform/crosvm/ \
```
Comment (andrisaar) [inline Dockerfile:241]: While you're in here cleaning up things, crosvm has updated its repository to https://chromium.googlesource.com/crosvm/crosvm; consider pulling in a newer revision from there.

Comment (tiziano88) [reply Dockerfile:241]: I tried naively but it didn't seem to work, created #3332 to track.

Comment (andrisaar) [reply Dockerfile:241]: Yeah, I don't know if the commit hashes survived the transition. You could just pick the latest revision from that new repository.

Comment (tiziano88) [reply Dockerfile:241]: I did try with the latest commit from that revision indeed, but that also didn't work.

**Review by conradgrobler — APPROVED** [review]

**Pull request merged**